src/Classes/PassiveSpecListControl.lua

```diff
@@ -19,25 +21,49 @@ local PassiveSpecListClass = newClass("PassiveSpecListControl", "ListControl", f
 		self:RenameSpec(newSpec, "Copy Tree", true)
 	end)
 	self.controls.copy.enabled = function()
-		return self.selValue ~= nil
+		return self.selValue ~= nil and #self.selections == 1
 	end
 	self.controls.delete = new("ButtonControl", {"LEFT",self.controls.copy,"RIGHT"}, 4, 0, 60, 18, "Delete", function()
-		self:OnSelDelete(self.selIndex, self.selValue)
+		self:OnSelDelete()
 	end)
+	self.controls.delete.tooltipText = "Use Ctrl-Left Click to multi select trees below"
 	self.controls.delete.enabled = function()
 		return self.selValue ~= nil and #self.list > 1
 	end
-	self.controls.rename = new("ButtonControl", {"BOTTOMRIGHT",self,"TOP"}, -2, -4, 60, 18, "Rename", function()
+	self.controls.convert = new("ButtonControl", {"LEFT",self.controls.delete,"RIGHT"}, 4, 0, 60, 18, "Convert", function()
+		if #self.selections > 0 then
+			-- sort and loop through the selection list backwards, as the ConvertSpec procedure inserts a new spec after the current
+			table.sort(self.selections)
+			for selId = #self.selections, 1, -1 do
+				local spec = self.list[self.selections[selId]]
+				if spec.treeVersion ~= latestTreeVersion then
+					treeTab:ConvertSpec(spec, selId)
+				end
+			end
+			-- Set the last spec clicked on as the current tree
+			self.treeTab:SetActiveSpec(self.selIndex)
+			if #self.selections ~= 1 then
+				main:OpenMessagePopup("Trees Converted", "You had selected "..#self.selections.." trees to be converted to "..treeVersions[latestTreeVersion].display..".\nNote that some or all of the passives may have been de-allocated due to changes in the tree.\n\nYou can switch back to the old tree using the tree selector at the bottom left.")
```

Comment:
I would change this message to "6 trees have been converted to [version]" to better match the message on the main tree view.